Saikilito/nodejs-postgis
# Pick map to add user

# Description
- In the Home component I defined a markerPosition state (subject to change) in which the information of the new user that is going to be added will be stored, I defined it in home since that information must be visible in Map and Sidebar.
- I also moved the hiddenUserForm state to Home, this to control that when the user clicks on the map it will direct him to his location, but if he wants to add a new user this click will add a temporary marker indicating the exact location where he will be added.
- The Marker component I renamed it to markers and it receives a prop results in which it will use the map method to render the markers, I did it this way so that no matter if there is data in results or not the Markers component will render and allow the user to click on the map.
- I changed the images so that the marker is not a truck but a person.
- I added new properties to the Input component, since now the user indicates the position of the new user by clicking on the map, it is not necessary to be able to modify the coordinates text fields

fixes # (issue)

### Screenshots
https://user-images.githubusercontent.com/82967045/227786964-a0522283-e878-4707-8c9e-f4fa5dcde78c.mp4

### Type of change




## How to test it



1. Go to Home
2. Click on the Add User button
3. Type the desired user name
4. You will notice that in the coordinate entries you cannot type (this is good).
5. With the add user form displayed click on the map (this saves the coordinates you have clicked).
6. Click on Add user to save the data.

## Checklist:

- [ ] Passes linters and tests, also is building.
